airflow/providers/presto/hooks/presto.py

```diff
@@ -276,7 +278,16 @@ def run(
             )
             sql = hql
 
-        return super().run(sql=self._strip_sql(sql), parameters=parameters, handler=handler)
+        if isinstance(sql, str):
+            sql = self.strip_sql_string(sql)
```

Comment:
Why do we need this here?
It's handled in dbapi hook isn't it?

Reply:
I moved `strip_sql_string()` to DbApiHook so probably we could move this operation to DbApiHook also